src/App.js

```diff
@@ -18,11 +18,14 @@ function App() {
       <Routes>
         <Route path="/" element={<Home />} />
         <Route path="/products" element={<ProductList />} />
-        <Route path="/profile" element={<Profile />} />
         <Route path="/login" element={<Login />} />
         <Route path="/signup" element={<Signup />} />
-        <Route path="/wishlist" element={<Wishlist />} />
-        <Route path="/cart" element={<Cart />} />
+        <Route path="/" element={<ProtectedRoutes />}>
```

Comment:
there are two index routes

Reply:
I only knew this way for adding protectedRoutes, also they are not the same routes the index is there just to guard the nested routes inside it.